LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2019-2020 Bisnode
```

Comment:
2021

Reply:
I put 2020-2021, first commit on this repo is from 2020 ;)